com.github.sdbg.debug.ui/src/com/github/sdbg/debug/DebugLaunch.java

```diff
@@ -0,0 +1,37 @@
+package com.github.sdbg.debug;
+
+import com.github.sdbg.debug.ui.internal.DartUtil;
+import com.github.sdbg.debug.ui.internal.chrome.ChromeLaunchShortcut;
+import com.google.gdt.eclipse.core.sdbg.IDebugLaunch;
+
+import org.eclipse.core.resources.IProject;
+import org.eclipse.core.resources.IResource;
+import org.eclipse.jdt.core.IJavaProject;
+import org.eclipse.jdt.core.JavaCore;
+
+public class DebugLaunch implements IDebugLaunch {
+
+  @Override
+  public boolean hasChromeLauncher() {
+    return true;
+  }
+
+  @Override
+  public void launchChrome(IProject project, String mode, String url) {
```

Comment:
Ditto. Let's change the name of this method to just launch(). 
Also, let's move the mode parameter to be the last one.